fast64_internal/oot/oot_utility.py

```diff
@@ -701,15 +701,15 @@ def setActorParameter(object, field, param, mask):
 
 def getLastElemIndex(actorKey, elemTag, flagType):
 	'''Looking for the last index of an actor's property (from XML data)'''
-	indexes = []
+	indices = []
 	for actorNode in root:
 		if actorNode.get('Key') == actorKey:
 			for elem in actorNode:
 				if elem.tag == elemTag:
 					if flagType is None or (flagType == 'Switch' and elem.get('Type') == flagType):
-						indexes.append(elem.get('Index'))
-	indexes.sort()
-	return f"{indexes[len(indexes) - 1]}" if len(indexes) != 0 else "0"
+						indices.append(int(elem.get('Index'), base=10))
+	indices.sort()
```

Comment:
is ``indices.sort()`` not needed at all? I used that because I thought it would be the fix to the issue where the indices from the XML aren't in the correct order

Reply:
max() gets the maximum, it's the same as sort() and take the last argument